docs/data-sources/confluent_ip_filter.md

```diff
@@ -0,0 +1,49 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "confluent_ip_filter Data Source - terraform-provider-confluent"
+subcategory: ""
+description: |-
+  
+---
+
+# confluent_ip_filter Data Source
+
+[![General Availability](https://img.shields.io/badge/Lifecycle%20Stage-General%20Availability-%2345c6e8)](https://docs.confluent.io/cloud/current/api.html#section/Versioning/API-Lifecycle-Policy)
+
+`confluent_ip_filter` describes an IP Group data source.
+
+-> **Note:** See [IP Filtering on Confluent Cloud](https://docs.confluent.io/cloud/current/security/access-control/ip-filtering/overview.html) for more details about the IP Filtering feature, its prerequisites, and its limitations.
+
+## Example Usage
+
+```terraform
+provider "confluent" {
+  cloud_api_key    = var.confluent_cloud_api_key    # optionally use CONFLUENT_CLOUD_API_KEY env var
+  cloud_api_secret = var.confluent_cloud_api_secret # optionally use CONFLUENT_CLOUD_API_SECRET env var
+}
+
+data "confluent_ip_filter" "example" {
+  id = "ipf-abc123"
+}
+
+output "example" {
+  value = data.confluent_ip_filter.example
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Argument Reference
+
+The following arguments are supported:
+
+- `id` - (Required String) The ID of the IP Group (e.g., `ipf-abc123`).
```

Comment:
```suggestion
- `id` - (Required String) The ID of the IP Filter (e.g., `ipf-abc123`).
```

Reply:
Nice find! I’ve gone ahead and applied your suggestion.